### ISSUE TITLE
Add controls in tab stripe to access tabs that are out of view

### ISSUE THREAD

**Issue by xandershelley** [Microsoft/vscode#39059]
- VSCode Version: 1.17
- OS Version: win7, 64 bit

Here are my tabs with a reasonably sized window.
![good](https://user-images.githubusercontent.com/13305268/33188747-169e9082-d051-11e7-9480-6244fec7a848.PNG)

And here is half sized window. 
![bad](https://user-images.githubusercontent.com/13305268/33188748-186e61b2-d051-11e7-8763-831cc124393c.PNG)

There needs to be some notepad++-like functionaly where there is an over arrow to access these inaccessable tabs. If this already exists I would be happy to delete this post, but I have yet to find any solution.

Reproduces without extensions: Yes

**Comment by usernamehw** [comment]
* commands `workbench.action.nextEditor` <kbd>Ctrl</kbd>+<kbd>PageDown</kbd> & `workbench.action.previousEditor` <kbd>Ctrl</kbd>+<kbd>PageUp</kbd>
* command quick pick `workbench.action.quickOpen` <kbd>Ctrl</kbd>+<kbd>P</kbd>
* command `workbench.action.quickOpenNavigateNextInEditorPicker` <kbd>Ctrl</kbd>+<kbd>Tab</kbd>
* mouse wheel scroll when cursor is over tabs bar
* click on ellipsis (top right) => Show opened editors
* close extra tabs

**Comment by xandershelley** [comment]
Those controls work, thanks!
Personally I still think there should be a visual control for this, since the only way to do it without knowing keyboard shortcuts (that I couldent find on google) is to resise the window itself which is a little rediculus. Should this be moved or removed?

**Comment by usernamehw** [comment]
IDK, seems like a valid issue. Wait for the assignee.

**Comment by nkernytskyy** [comment]
> mouse wheel scroll when cursor is over tabs bar
-- is awesome!!

**Comment by mfgray** [comment]
I often work with the screen split vertically and a few source files open that I go between. Currently there appears to be no convenient or self evident way to view the names of and select the open tabs since some of them are completely invisible if there is not enough screen space.

I consider switching to sublime over the issue, so it's a pretty important usability requirement and I feel it was underestimated closed. Can we get this ticket added to the roadmap?

**Comment by mikepeiman** [comment]
Agreed, to reopen this. While this is not a crucial/deal-breaker issue, it is important: regular-use UI is not trivial. 

I did not even realize there was a horizontal slider until I came to look into this issue. So, there is - but it's ridiculously narrow. No user should ever be tasked to click on something a couple of pixels wide in a utility context (if it's a game, for some purposes ok....). 

Simplest solution: make this a FAT scrollbar on hover with some color so it is easily visible AND clickable.
